sklearn/semi_supervised/tests/test_label_propagation.py

```diff
@@ -143,6 +144,29 @@ def test_sparse_input_types(
     assert_array_equal(clf.predict([[0.5, 2.5]]), np.array([1]))
 
 
+@pytest.mark.parametrize("constructor", CONSTRUCTOR_TYPES)
+@pytest.mark.parametrize("Estimator, parameters", ESTIMATORS[:2])
```

Comment:
this `ESTIMATORS[:2]` is brittle. We might change that list in the future, and I'm not sure why not all of them are tested here.

Reply:
I think it's testing `LabelPropagation` instances only. Maybe creating a new constant:
```python
LP_ESTIMATORS = [est for est in ESTIMATORS if isinstance(est, LabelPropagation)]
```